packages/experimental/Shimmer/src/Shimmer.styling.ts

```diff
@@ -6,11 +6,13 @@ import { defaultShimmerTokens } from './ShimmerTokens';
 
 export const stylingSettings: UseStylingOptions<ShimmerProps, ShimmerSlotProps, ShimmerTokens> = {
   tokens: [defaultShimmerTokens, shimmerName],
+  tokensThatAreAlsoProps: 'all',
   slotProps: {
     root: buildProps(
       (tokens: ShimmerTokens) => ({
         accessibilityRole: 'progressbar',
         accessible: true,
+        style: { overflow: 'hidden', backgroundColor: tokens.backgroundColor },
```

Comment:
Seems Shimmer has only one slot as 'root'. 
Shouldn't we add this directly token files would be good idea ?  any specific reason to add it here 🤔